### PR TITLE
Remove `import dask as d` from docstrings

### PR DESCRIPTION
**Pull request by mrocklin**
This seems strange and is not a recommended import name

### CODE REVIEW

**Review by rjzamora — APPROVED** [review]
Thanks @mrocklin !

File: dask/base.py [hunk 1]
```diff
@@ -568,7 +568,7 @@ def compute(
 
     Examples
     --------
-    >>> import dask as d
+    >>> import dask
```
Comment (rjzamora) [inline dask/base.py:571]: Looks like there is a `>>> d.compute(a, b)` line below that needs to change to `>>> dask.compute(a, b)`

**Review by rjzamora — APPROVED** [review]
gpuci failures are unrelated.

**Review by jrbourbeau — APPROVED** [review]
Thanks @mrocklin

**Pull request merged**